SPECS/rubygem-bigdecimal/rubygem-bigdecimal.spec

```diff
@@ -14,7 +14,7 @@ Source0:        https://github.com/ruby/bigdecimal/archive/refs/tags/v%{version}
 BuildRequires:  git
 BuildRequires:  ruby
 Requires:       ruby(release)
-Provides:       bigdecimal = %{version}-%{release}
+Provides:       rubygem-bigdecimal = %{version}-%{release}
```

Comment:
Do we need these `rubygem-%{gem_name}` provides, given that we use that name for the RPM package itself?

Reply:
Honestly, I am not super sure, but I am adding it to save a potential pipeline failure

Reply:
Ok, removing the generic ones, sorry for the back and forth

Reply:
Ah, you're right, @oliviacrain, I missed the fact that now the packages themselves have the correct names, so only the `rubygem([name])` may be necessary. Or a generator for Ruby packages but that's for a different day.;)